### ISSUE TITLE
Cannot read property 'val' of null

### ISSUE THREAD

**Issue by randomname32** [ioBroker/ioBroker.lovelace#213]
**Describe the bug**  
lovelace instance crashes directly after startup.

While playing around with the visualization and renaming iobroker objects at the same time the lovelace installation got into a messed up state (see attached logfile output). Uninstalling and reinstalling the adapter did not help.

**Expected behavior**  
- check for "null" before trying to access "val"
- uninstalling/reinstalling should ideally provide a fresh&clean working solution, i.e. previous configuration files should be deleted (I don't know if this might be the issue here)

**Screenshots & Logfiles**  

host.raspberrypi | 2021-06-07 00:13:22.760 | error | instance system.adapter.lovelace.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
-- | -- | -- | --
lovelace.0 | 2021-06-07 00:13:22.079 | warn | (9584) Terminated (UNCAUGHT_EXCEPTION): Without reason
lovelace.0 | 2021-06-07 00:13:22.075 | info | (9584) terminating
lovelace.0 | 2021-06-07 00:13:22.068 | info | (9584) cleaned everything up...
lovelace.0 | 2021-06-07 00:13:22.026 | error | (9584) Cannot read property 'val' of null
lovelace.0 | 2021-06-07 00:13:22.024 | error | at processImmediate (internal/timers.js:463:21)
lovelace.0 | 2021-06-07 00:13:22.024 | error | at Immediate.&lt;anonymous&gt; (/opt/iobroker/node_modules/iobroker.lovelace/lib/server.js:2087:88)
lovelace.0 | 2021-06-07 00:13:22.024 | error | (9584) TypeError: Cannot read property 'val' of null
lovelace.0 | 2021-06-07 00:13:22.018 | error | (9584) uncaught exception: Cannot read property 'val' of null
lovelace.0 | 2021-06-07 00:13:20.466 | info | (9584) http server listening on port 8091
lovelace.0 | 2021-06-07 00:13:20.278 | info | (9584) starting. Version 1.0.16 in /opt/iobroker/node_modules/iobroker.lovelace, node: v12.22.1, js-controller: 3.2.16

**Versions:**  
 - Adapter version: 1.0.16
 - JS-Controller version: 3.2.16
 - Node version: 12.22.1
 - Operating system: raspbian

**Comment by Garfonso** [comment]
I will look into this. But "val of null" hints in the direction that a state (of any entity, probably manual one) is null, ie there is no value shown in admin. For a work around until I fix this, you could try to find something like this.

**Comment by randomname32** [comment]
Thank you Garfonso for your quick reply!
Your hint helped me to fix the problem. I manually went through all objects that I had configured to work with lovelace and deactivated them for lovelace. 

Now I have an empty configuration/visualisation and can start new. This is fine for me, since I have just recently started using iobroker and lovelace and my visualization was more or less a playground. From now on I will create backups of the configuration from time to time...

As additional information for others who might run into this problem:
While the instance was running (or constantly restarting), I was able to do this by clicking on the wrench icon and unticking the lovelace checkbox. Without lovelace running I had to do this via the "pen" icon and deleting the json entry.
